homeassistant/components/bmw_connected_drive/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "bmw_connected_drive",
   "name": "BMW Connected Drive",
   "documentation": "https://www.home-assistant.io/integrations/bmw_connected_drive",
-  "requirements": ["bimmer_connected==0.7.21"],
+  "requirements": ["bimmer_connected==0.8.1"],
```

Comment:
In 0.8.1 `pycryptodome` was added as a requirement and pinned to the exact version 3.11.0. We want to avoid that libraries pin requirements to an exact version to make the libraries more compatible with other libraries in Home Assistant that have the same requirements. Please loosen the pinning and require only a minimum version or a range with as forgiving limits as possible.

Thanks!

Reply:
Done. Of course that makes a lot of sense!

Reply:
Oops, test fail as `bimmer_connected==0.8.2` cannot be downloaded. But it has been released to PyPi: https://pypi.org/project/bimmer-connected/#history

Reply:
I restarted CI.